app/components/NewsletterSignup.tsx

```diff
@@ -48,8 +48,8 @@ export default function NewsletterSignup() {
             <div className="w-fit flex justify-center mt-8 mb-12 mx-3">
               <input
                 type="text"
-                placeholder="Enter your email address"
-                className="w-fit text-left md:text-center border border-luskin-blue rounded-[5px] pt-2 pb-3 px-3 md:px-6 placeholder:opacity-50 placeholder:text-black placeholder:text-[0.92rem] md:placeholder:text-base"
+                placeholder="Enter your email"
+                className="w-fit text-center md:text-center border border-luskin-blue rounded-[5px] pt-2 pb-3 px-3 md:px-6 placeholder:opacity-50 placeholder:text-black placeholder:text-[0.92rem] md:placeholder:text-base"
```

Comment:
Looks like adding text-center causes the cursor to also be aligned at the center - usual practice is that it starts left aligned.